bitnami/kubeapps/values.yaml

```
@@ -1843,6 +1843,157 @@ kubeappsapis:
     automountServiceAccountToken: true
     annotations: {}
 
+## @section OCI Catalog chart configuration
+ociCatalog:
+  ## @param ociCatalog.enabled Enable the OCI catalog gRPC service for cataloging
+  ## OCI repositories
+  enabled: false
+  ## Bitnami Kubeapps OCI Catalog image
+  ## ref: https://hub.docker.com/r/bitnami/kubeapps-ocicatalog/
+  ## @param ociCatalog.image.registry OCI Catalog image registry
+  ## @param ociCatalog.image.repository OCI Catalog image repository
+  ## @param ociCatalog.image.tag OCI Catalog image tag (immutable tags are recommended)
+  ## @param ociCatalog.image.digest OCI Catalog image digest in the way sha256:aa.... Please note this parameter, if set, will override the tag
+  ## @param ociCatalog.image.pullPolicy OCI Catalog image pull policy
+  ## @param ociCatalog.image.pullSecrets OCI Catalog image pull secrets
+  ## @param ociCatalog.image.debug Enable image debug mode
+  ##
+  image:
+    registry: docker.io
+    repository: bitnami/kubeapps-oci-catalog
+    tag: latest
```

```suggestion
    tag: 2.9.0-debian-11-r0
```

Adding what will most likely be the tag for this image (only - the other images aren't set in this PR, and will be set by the release)